<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Setting</class>
 <widget class="QDialog" name="Dialog_Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_Setting{border-image: url(:/images/bkg41.png);}
QPushButton{
    color: white;
	background-color: rgb(53, 63, 75);
}
QRadioButton{
    color: white;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Data">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>读取规划数据</string>
   </property>
   <widget class="QPushButton" name="pushButton_Femur_AP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>股骨正位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Femur_Lat">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>股骨侧位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tibia_AP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>胫骨正位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tibia_Lat">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>胫骨侧位</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>UDP连接</string>
   </property>
   <widget class="QPushButton" name="pushButtonUDP">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_UDP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>5555</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_TypeofNDI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NDI</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_TypeofTracker">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>117</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracker</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>初始化Marker名称</string>
   </property>
   <widget class="QLabel" name="label_port_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>股骨：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_NameFemur">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TTBLOCK0714</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_NameTibia">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker_UR_new</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>胫骨：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_NameXspot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>XSPOT1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Xspot：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_NameTipTool">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MARKERTIP_LONG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>探针：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_InitMarker">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>机器人：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_NameRobot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker_Robot</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdi_NameXspot_2">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>90</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>XSPOT1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>641</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>NDI</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Femur">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resources/resource/Femur_disable.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_Tibia">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resources/resource/Tibia_disable.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_Xspot">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resources/XSpot_disable.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_Tip">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resources/tip_disable.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_Robot">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resources/ur_robot_disable.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
